### ISSUE TITLE
Unzipper library corrupting files in Node.js v19.8.1

### ISSUE THREAD

**Issue by webwake** [neutralinojs/neutralinojs-cli#200]
The unzipper library causes extracted files to get corrupt in the latest Node.js v19.8.1 on Windows 10 (worked with Node v18, have not tested other versions)

Also created an [issue in unzipper](https://github.com/ZJONSSON/node-unzipper/issues/271), although library appears to be abandoned 

May wish to switch to [decompress library](https://www.npmjs.com/package/decompress) as that appears to be working

**Comment by nicorac** [comment]
Same here: Windows 10 22H2 x64 + Node 18.16.0

Just created a [pull request](https://github.com/neutralinojs/neutralinojs-cli/pull/204) to replace `unzipper` with `decompress` which works correctly.

I haven't published it on npm so, if anyone wants to test it, just install from my forked repo:

```
npm remove -g @neutralinojs/neu
npm install -g https://github.com/nicorac/neutralinojs-cli.git
```

**Comment by shalithasuranga** [comment]
Hey.. I am closing this one since the issue was fixed via #204 Thanks all :)
